Source/Core/Core/FifoPlayer/FifoDataFile.cpp

```diff
@@ -72,6 +72,11 @@ FifoDataFile::FifoDataFile() = default;
 
 FifoDataFile::~FifoDataFile() = default;
 
+bool FifoDataFile::ShouldGenerateFakeVIUpdates() const
+{
+  return m_Version <= 4;
```

Comment:
I don't think this is needed, until someone adds a FIFO log version that _doesn't_ need fake VI updates.